### ISSUE TITLE
False positive: unecessary_string_escapes 

### ISSUE THREAD

**Issue by mfizz1** [passsy/dart-lint#66]
```dart
RegExp(
  r'(\s)'
  '?(foo|bar)(\s)?',
  caseSensitive: false,
),
```
the third matching group `(\s)` is incorrectly identified as `unecessary_string_escapes`

**Comment by passsy** [comment]
The warning is correct. You miss the `r` in the second line

```dart
    RegExp(
      r'(\s)'
      r'?(foo|bar)(\s)?',
      caseSensitive: false,
    );
```

**Comment by mfizz1** [comment]
My mistake! I thought with the line continuation, the `r` from the first line would have been sufficient.
